### ISSUE TITLE
Can't use this script with Greasemonkey v4

### ISSUE THREAD

**Issue by UnicDuckEli5** [bberenz/sgtfrog#54]
.

**Comment by akberenz** [comment]
I am aware of this, unfortunately GMv4 is horrible mess at the moment, so even if I were to make it compatible, a lot of functionality would be missing. It is on my scope, but likely won't be handled soon.

In the mean time, both [ViolentMonkey](https://addons.mozilla.org/en-US/firefox/addon/violentmonkey/) and [TamperMonkey](https://addons.mozilla.org/en-US/firefox/addon/tampermonkey/) are available for FF57, and are compatible with any existing scripts from GMv3 including this one.

It's possible to transfer all your scripts over to either of these alternative, via [this guide](https://stackoverflow.com/questions/47317983/how-to-transfer-all-greasemonkey-userscripts-to-tampermonkey-on-firefox-57), should you need to. (Instructions are roughly the same for violentmonkey, just "Settings" instead of "Dashboard")
